packages/lesswrong/components/posts/PostsPage/PostsPageEventData.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { registerComponent, Components } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles'
+import withUser from '../../common/withUser'
+
+const styles = theme => ({
+  metadata: {
+    marginTop:theme.spacing.unit*3,
+    ...theme.typography.postStyle,
+    color: 'rgba(0,0,0,0.5)',
+    fontSize: "1.2rem",
```

Comment:
Hmm, would obviously prefer to keep font-size proliferation to a minimum, but might be inevitable.